rdrf/report/schema.py

```diff
@@ -190,6 +191,14 @@ def resolve_gender(parent_guardian, _info):
         return dict(ParentGuardian.GENDER_CHOICES).get(parent_guardian.gender, parent_guardian.gender)
 
 
+class EmailNotificationsType(ObjectType):
+    unsubscribe_all = graphene.Boolean()
+
+    def resolve_unsubscribe_all(patient, _info):
```

Comment:
As long as you change the resolver to resolve the EmailPreference, you can remove the lookup here and replace the `patient` parameter with the `emailPreference` object